src/main/java/org/opentripplanner/util/DateUtils.java

```diff
@@ -282,4 +282,11 @@ public static long absoluteTimeout(double relativeTimeoutSeconds) {
         else
             return System.currentTimeMillis() + (long)(relativeTimeoutSeconds * 1000.0);
     }
+
+    /**
+     * Returns false if the given epochSeconds is more than ttlMinutes in the past.

```

Comment:
Spell out time-to-live minutes.

Reply:
fixed in 0b5b987b035c15264f45fc1eaf1e0cc5979fbe38